package.json

```
@@ -58,7 +58,9 @@
     "node-emoji": "^1.11.0",
     "node-pty": "^0.10.1",
     "prompt": "^1.2.0",
-    "term.js": "^0.0.7"
+    "term.js": "^0.0.7",
+    "ts-node": "^10.2.1",
+    "typescript": "^4.4.3"
```

Fix versions for all packages.
```suggestion
    "term.js": "0.0.7",
    "ts-node": "10.2.1",
    "typescript": "4.4.3"
```